spinoso-string/src/enc/impls.rs

```diff
@@ -0,0 +1,204 @@
+//use alloc::borrow::Cow;
+use alloc::vec::Vec;
+use core::borrow::{Borrow, BorrowMut};
+use core::fmt::Arguments;
+use core::ops::{Deref, DerefMut};
+#[cfg(feature = "std")]
+use std::io::{IoSlice, Result, Write};
+
+use super::EncodedString;
+
+#[cfg(feature = "std")]
+impl Write for EncodedString {
+    #[inline]
+    fn write(&mut self, buf: &[u8]) -> Result<usize> {
+        match self {
+            EncodedString::Ascii(n) => n.write(buf),
```

Comment:
Sorry to be nit-picky, but can we replace all uses of `n` for the inner bit of the enum with `inner`?